tests/common/devices.py

```diff
@@ -213,19 +213,23 @@ def _get_npu_count(self, platform):
         """
         Gets the number of npus for this device.
         """
-
+        num_npu = 1
```

Comment:
@SuvarnaMeenakshi  Good opportunity to make make npu asic.
We have to fix that in all places going forward.

Reply:
Updated as suggested.